### DISCUSSION TITLE
Status of qt6 + lxqt?

### DISCUSSION THREAD

**Discussion by rubyFeedback** [lxqt/lxqt#2616] [General]
Hey folks,

I am having build issues with qt5; in the past the LFS/BLFS project helped me compile it, but they semi-phased out qt5, and qt6 actually compiles for me without problem. So as I don't need qt5 that much, save for KDE's konsole, I decided a while ago to switch to qt6. Naturally not all "downstream" code is qt6-ready yet.

Are there any plans for lxqt to transition into qt6? If so when, very roughly speaking, might this be happening? Right now I am a bit in limbo because I no longer use qt5 really.

**Comment by tsujan** [comment]
> Are there any plans for lxqt to transition into qt6?

LXQt was ported to Qt6 in its version 2.0.0, in April.
